examples/bri-3/src/bri/zeroKnowledgeProof/api/err.messages.ts

```diff
@@ -1,4 +1,5 @@
 export const INVALID_ANCHOR_HASH_INPUT =
   'Input document provided for anchor hash does not exist or is blank';
+
 export const CCSM_ANCHOR_HASH_NOT_FOUND_ERR_MESSAGE =
-  'Anchor Hash does not exist on the CCSM.';
+  'Anchor Hash does not exist on the CCSM or the local storage.';
```

Comment:
not important for now probably but should be split into 2 error messages and 2 error cases maybe for easier debugging?